src/pyfmi/fmi_algorithm_drivers.py

```diff
@@ -131,6 +131,12 @@ class AssimuloFMIAlgOptions(OptionBase):
             This MUST be provided.
             Default: None
 
+        result_max_size --
+            Maximum size of the stored result (in bytes). This is not a hard limit, the
+            actual size will be slightly larger to account for that the result need to
+            be consistent.
+            Default: 2GB
```

Comment:
I think we should write:
`Default: 1024**3*2 (2GB)`

Reply:
Updated.